pkg/config/builder_test.go

```diff
@@ -297,12 +302,18 @@ func TestLoadFile(t *testing.T) {
 	l := NewBuilder()
 	l.File(file.Name())
 
+	mutex := sync.Mutex{}
 	for _, test := range configsTest {
+		// Lock this loop
+		mutex.Lock()
+		defer mutex.Unlock()
```

Comment:
Now you need to explain me why we need to acquire the lock :-) .

Reply:
well we don't ... looking into why we can't use defer ...